tests/director/api/runtime_api_test.go

```diff
@@ -130,14 +130,15 @@ func TestRuntimeRegisterUpdateAndUnregister(t *testing.T) {
 
 	// update runtime, check if only simple values are updated
 	//GIVEN
+	runtimeStatusCond := graphql.RuntimeStatusConditionConnected
 	givenInput.Name = "updated-name"
 	givenInput.Description = ptr.String("updated-description")
 	givenInput.Labels = &graphql.Labels{
 		"key": []interface{}{"values", "aabbcc"},
 	}
+	givenInput.StatusCondition = &runtimeStatusCond
```

Comment:
We can put the creation and usage together

Reply:
Not really, you cannot take the address of a constant value, you need to go through the variable assignment.

Reply:
I mean:
```
runtimeStatusCond = graphql.RuntimeStatusConditionConnected
givenInput.StatusCondition = &runtimeStatusCond
```